source/jormungandr/jormungandr/georef.py

```diff
@@ -83,6 +83,8 @@ def get_crow_fly(
         filter=None,
         stop_points_nearby_duration=300,
         request_id=None,
+        forbidden_uris=None,
+        allowed_id=None,
```

Comment:
Having for default value an empty array would make your code much simpler
```suggestion
        forbidden_uris=[],
        allowed_id=[],
```